core/src/datayoga_core/result.py

```diff
@@ -2,18 +2,22 @@
 from enum import Enum
 from typing import Any, Dict, List, Optional
 
-Status = Enum("Status", "SUCCESS REJECTED FILTERED")
+
+class Status(Enum):
+    SUCCESS = "SUCCESS"
+    REJECTED = "REJECTED"
+    FILTERED = "FILTERED"
```

Comment:
Shouldn't it be something like this:
```
@unique
class DbType(str, Enum):
    """
    Defines supported databases
    """
    CASSANDRA = "cassandra"
    MYSQL = "mysql"
    ORACLE = "oracle"
    POSTGRESQL = "postgresql"
    SQLSERVER = "sqlserver"
```

So you can access to a value without .value?

Reply:
✅